tests/openebs-pool-container-failure_test.go

```diff
@@ -164,7 +164,6 @@ var _ = Describe("BDD of openebs pool container failure experiment", func() {
 			//Fetching the runner pod and Checking if it get in Running state or not
 			By("Updating the result table")
 			By("Wait for chaso-runner to come in running sate")
-			pipelineResult, err := utils.ResultUpdate(experimentName, engineName, clientSet)
```

Comment:
This is not required?

Reply:
typo

Reply:
Why are we not updating result table for this experiment?

Reply:
@ispeakc0de  pls see line 249